template.feature

```diff
@@ -1 +1,163 @@
 # This is the main template file containing all feature interactions# This is the main template file containing all feature interactions# This is the main template file containing all feature interactions
+
+INTERACTIONS:
+
+Button:
+check if button exists
+	I should see a "<label>" button <context> "<value>"
+	Examples:
+	| context    |
+	| in section |
+	| with id    |
+	eg: I should see a "Confirm" button with id "profile-form-confirm"
+	I should see a "Confirm" button in section "profile-form"
+
+click a button
+I select the "<label>" button <context> "<value>"
+I click on the "<label>" button <context> "<value>"
+	Examples:
+	| context    |
+	| in section |
+	| with id    |
+	eg: I select the "Confirm" button with id "profile-form-confirm"
+	I click on the "Confirm" button with id "profile-form-confirm"
+
+check if disabled/enabled
+I should see the "<label>" button <context> "<value>" is enabled|disabled
+	Examples:
+	| context    |
+	| in section |
+	| with id    |
+	eg: I should see the "Confirm" button with id "profile-form-confirm" is enabled
+	I should see the "Confirm" button in section "profile-form" is disabled
+
+check style(color)
+I should see the "<label>" button <context> "<value>" has <style> "<style_value>"
+	Examples:
+	| context    | style   |
+	| in section | color   |
+	| with id    | opacity |
+#id format: page-section-description
+	eg: I should see the "Confirm" button with id "profile-form-confirm" has color "green"
+
+
+Text:
+check if text exists ( I should see )
+I should <visibility> "<string>"
+	Examples:
+	| visibility |
+	| see        |
+	| not see    |
+eg: I should not see "Today is Monday"
+
+click on text
+I click on the "<label>" text
+	eg: I click on the "Confirm" text
+
+text position check
+I should see "<string>" <position> "<string>"
+	Examples:
+	| position  |
+	| preceding |
+	| following |
+eg. I should see "hello" preceding "how are you"
+
+
+Textbox:
+Enter text in textbox
+I enter "<string>" in textbox <context> "<value>"
```

Comment:
add a field for textbox name. 

`I enter "<string>" in "<textbox_name>" <context> "<value>"`